tests/image_gen/cs3_screens.py

```diff
@@ -1,5 +1,20 @@
+calledStart = False
+calledActivated = False
+
+def a_onScreenStart(app):
```

Comment:
Does this test capture the requirements for onScreenActivated? All screens' onScreenStart functions are called when runAppWithScreens is called, then activeScreen_onScreenActivated is called, and then activeScreen_redrawAll. 

After that, newScreen_onScreenActivated is called whenever setActiveScreen('newScreen') is called.